tests/atomic/main.yml

```diff
@@ -415,7 +415,7 @@
     # Remove the locally tagged image so we can pull it again from the private
     # registry
     - role: docker_rmi
-      drmi_image: "{{ local_name }}:latest"
+      drmi_image_name: "{{ local_name }}:latest"
```

Comment:
It looks like the docker_rmi role does not like the image name with tag being passed in because it looks for the whole string in the docker images output and the tag is separated in the docker images output.

```

TASK [docker_rmi : Fail if the image is not present] ***************************
fatal: [testnode]: FAILED! => {
    "changed": false, 
    "failed": true
}

MSG:
---
Expected: 172.17.0.1:5000/alpine:latest container is in docker images
Actual: docker images REPOSITORY               TAG                 IMAGE ID            CREATED             SIZE
docker.io/alpine         latest              dca2bcc01afd        1 seconds ago       4.143 MB
docker.io/busybox        latest              5b0d59026729        2 weeks ago         1.146 MB
docker.io/registry       2                   d1fd7d86a825        4 weeks ago         33.26 MB
172.17.0.1:5000/alpine   latest              3fd9065eaf02        4 weeks ago         4.143 MB
```

Reply:
Totally my fault, I didn't actually test these changes on the `atomic` test.  :\

I'll get it fixed tomorrow, likely.

Reply:
The robots do the testing.  I just looked at the CI failure :laughing: